pkg/deploy/deploy.go

```diff
@@ -43,13 +43,13 @@ type deployer struct {
 	log *logrus.Entry
 	env env.Core
 
-	globaldeployments      features.DeploymentsClient
-	globalgroups           features.ResourceGroupsClient
-	globalrecordsets       dns.RecordSetsClient
-	globalaccounts         storage.AccountsClient
-	deployments            features.DeploymentsClient
-	groups                 features.ResourceGroupsClient
-	loadbalancers          network.LoadBalancersClient
+	globaldeployments features.DeploymentsClient
+	globalgroups      features.ResourceGroupsClient
+	globalrecordsets  dns.RecordSetsClient
+	globalaccounts    storage.AccountsClient
+	deployments       features.DeploymentsClient
+	groups            features.ResourceGroupsClient
+	// loadbalancers          network.LoadBalancersClient
```

Comment:
We should just remove this entirely I think

Reply:
remove what? line 52 `// loadbalancers          network.LoadBalancersClient` ?
I was not sure about if this is affecting something or not, so I just commented it out for now. I will remove that as well once I have more peer reviews on this.

Reply:
do we have enough data now to remove this? What data would we need to make a decision to keep/remove this comment?